src/services/at-sce-api.service.js

```diff
@@ -22,9 +24,18 @@ const AT_SCE_SERVICE_URI = process.env.AT_SCE_SERVICE_URI;
 console.log(`AT_SCE_SERVICE_URI: ${AT_SCE_SERVICE_URI}`);
 
 // Operation: Login - POST /api/v1/login
-AT_SCE_SERVICE.login = () => {
-  return {};
-};
+AT_SCE_SERVICE.login = (data) => {
+  return axios({
+      method: "POST",
+      url: `${AT_SCE_SERVICE_URI}/v1/login`,
+      data: data,
+      headers: {
+        "content-type": "application/json",
+      },
+    }).catch(function (error) {
+      console.log("Error: " + error.message);
```

Comment:
you can use template literals here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals